frontend/server/router.ts

```diff
@@ -188,43 +187,7 @@ export const appRouter = router({
       const res = await s3Upload(filePath)
       return res;
     }),
-  getDistinctId: publicProcedure
```

Comment:
This function is still referenced in the frontend, can you remove where it's referenced?

Reply:
done